portality/scripts/journals_update_via_csv.py

```diff
@@ -82,6 +82,7 @@
             reader = csv.DictReader(g, fieldnames=header_row)
 
             # verify header row with current CSV headers, report errors
+            # TODO: Include 'Owner' field - but we should probably base this process off the AdminCSV too.
```

Comment:
This is a TODO to explain that we're further diverging from the outgoing CSV here, by including Owner as a check. It'll still be accepted (unmapped) but if we're formalising this process in the future, e.g. https://github.com/DOAJ/doajPM/issues/3619, we should use the Admin CSV and expect the additional columns to be unmodified